src/controllers/logoutController.ts

```diff
@@ -0,0 +1,40 @@
+import { Blacklist } from "../database/models/blacklist";
+import { HttpException } from "../utils/http.exception";
+import { Request, Response } from "express";
+import { v4 as uuidv4 } from "uuid";
+
+const logout = async (req: Request, res: Response) => {
+	try {
+		const token = req.header("Authorization")?.split(" ")[1];
+
+		if (!token) {
+			return res
+				.status(404)
+				.json(new HttpException("NOT FOUND", "Token Not Found"));
+		}
+
+		const blacklistedToken = await Blacklist.findOne({ where: { token } });
+
+		if (!blacklistedToken) {
+			await Blacklist.create({ id: uuidv4(), token: token });
```

Comment:
Imagine here we have 10k users one user logout at least 3 times in day imagine number of blacklisted token we are gone to have in our database. after one year. Find way to delete token in blacklist table.

Reply:
Find way to delete token in blacklist table.